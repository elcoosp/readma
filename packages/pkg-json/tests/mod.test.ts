import { assertSnapshot } from '@std/testing/snapshot'
import { loadPkgJson } from '../src/mod.ts'
Deno.test('mod', async (t) => {
  await assertSnapshot(t, await loadPkgJson('.'))
})
