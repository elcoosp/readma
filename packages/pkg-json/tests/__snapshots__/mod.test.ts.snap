export const snapshot = {};

snapshot[`mod 1`] = `
{
  description: "{{description}}",
  files: [
    "src",
  ],
  name: "@readma/pkg-json",
  publishConfig: {
    access: "public",
  },
  scripts: {
    "test:cov": "deno test -A --coverage=cov/ && deno coverage --lcov cov/ > cov.lcov",
    "test:up": "pnpm test -- -- --update",
    "test:watch": "deno test --watch",
    build: \`echo "Nothing to build, it's deno!"\`,
    check: "deno check src/mod.ts",
    dev: "deno run --watch src/mod.ts",
    doc: "deno doc --html src/mod.ts",
    fmt: "deno fmt",
    lint: "deno lint",
    test: "deno test -A",
  },
  type: "module",
  version: "0.0.0",
}
`;
